### ISSUE TITLE
Multiple instances support

### ISSUE THREAD

**Issue by mobileben** [pixijs/pixi-sound#120]
This is part question, perhaps a feature request depending. 

I typically build games in C++ using engines I've built. My audio/sound managers have allowed for multiple instances of a given sound effect. The managers typically allocate the audio buffer and associates specific instances to the buffer (and manages the playback if necessary).

Taking a look at the `pixi-sound` code I do see some things regarding multiple instances. But looking at aliases and how things are laid out, it seems that I would perhaps need to do a bit more to get multiple instances of a sound working?

Is the proper way of creating multiple instances of an asset to associate a different alias for that asset? For example:

'test1' associated with 'assets/test.mp3'
'test2' associated with 'assets/test.mp3'

At least with this stuff I've used in the past, a `AudioBuffer` or `SoundBuffer` would be created for an asset (eg. `test` associated with the buffer for `assets/test.mp3`. I would then just need to create my `Sound` object from the `test` `SoundBuffer`.

So it would look something like this (C++ syntax).

```
Sound *sfx1 = SoundManager.createSfx("test");
Sound *sfx2 = SoundManager.createSfx("test");
```

Currently I preload though the loader at the start of the game along with my textures and other assets. When doing this, I only preload an `mp3` once with a "common name" such as 'test'. I was hoping to, much like my C++ reference code above, create multiple instances of the mp3 to play.

The way I am getting the asset right now is through the `sound` property from `app.loader.resources['test'].sound`. This gives me direct access to `PIXI.sound.Sound` which I want, because I want sound objects I can reuse. 

From what I can tell, however, is that this is in actuality only one instance of a sound object.  So, for example, if I played it "twice" with a small time gap, it would interrupt itself versus me hearing 2 slightly separate playbacks (like two overlapping gun shots).

What is the proper way to do multiple instances?

I'll no doubt have followups on this I think.

**Comment by djmisterjon** [comment]
> What is the proper way to do multiple instances?

On my side it work for multiple instance
`PIXI.sound._sounds[id].play()`
Multimple time and i get overlas
![image](https://user-images.githubusercontent.com/24865815/70415392-80cba600-1a2a-11ea-9c1d-eb28421bf7c4.png)
On my side  ` $audio === PIXI.sound`.
Am not sure why your instance, get interrupt itself on small gaps !

**Comment by mobileben** [comment]
Ahh, okay. I see what is happening. This is my bad and misinterpretation of things. 

I also reviewed the pixi-sound code a bit more regarding this and see that Sound actually tracks instances of itself playing in a pool.

The effect I was testing with sounded like it was cut out, but it wasn't. I added a bit more debug around it and used a diff sound and it was more apparent it was behaving as I would have thought.

Thanks for answering this.
